### PR TITLE
add period service on frontend

### PR DESCRIPTION
**Pull request by TumkoVolodymyr**
https://trello.com/c/qA2Xok98

### CODE REVIEW

File: frontend/src/services/periodService.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+const PERIOD_TODAY = 'today';
+const PERIOD_YESTERDAY = 'yesterday';
+const PERIOD_LAST_WEEK = 'last_week';
+const PERIOD_LAST_MONTH = 'last_month';
+const PERIOD_LAST_QUARTAL = 'last_quartal';
+const PERIOD_ALL = 'all_period';
+
+export const DAYS_IN_WEEK = 7;
```
Comment (TatsuUkraine) [inline frontend/src/services/periodService.js]: not quite sure that you need to export all this con values

**Comment by TatsuUkraine** [comment]
@TumkoVolodymyr issue still remains - all this service can be simplified a lot if you use momentjs, if not - let me know

**Comment by TumkoVolodymyr** [comment]
> @TumkoVolodymyr issue still remains - all this service can be simplified a lot if you use momentjs, if not - let me know

@TatsuUkraine  can we left as it is for this time?

**Comment by TatsuUkraine** [comment]
> > @TumkoVolodymyr issue still remains - all this service can be simplified a lot if you use momentjs, if not - let me know
> 
> @TatsuUkraine can we left as it is for this time?

@TumkoVolodymyr for what time?) this will never be changed if commit it like this) try to make appropriate implementation without reinventing a wheel. If you get any problems - let me know. I would also recommend to implement period service in separate branch, so if you stuck with it and everyone else will be block - we could merge this variant

**Pull request closed without merging**